week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
```

Comment:
This passes the tests, which is great :) Unfortunately, the tests were wrong!! (Sorry! Though this happens in real life too, it's really inconvenient!)

Please you update the last three lines of this file to instead be:
```javascript
test("formatCurrency function - case 1 works", formatCurrency(15) === "£18.00")
test("formatCurrency function - case 2 works", formatCurrency(17.5) === "£21.00")
test("formatCurrency function - case 3 works", formatCurrency(34) === "£40.80")
```

and then make them pass? If you read through the comment above, it mentions something about decimal places you'll need to make work :)